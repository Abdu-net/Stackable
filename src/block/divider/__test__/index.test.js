import { edit, save } from '../index'

describe( 'Divider Block', () => {
	test( 'block edit matches snapshot', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				height: 1,
				width: 50,
				color: '#dddddd',
				alignment: 'center',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( 'block save matches snapshot', () => {
		const wrapper = save( {
			attributes: {
				height: 1,
				width: 50,
				color: '#dddddd',
				alignment: 'center',
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )
} )
