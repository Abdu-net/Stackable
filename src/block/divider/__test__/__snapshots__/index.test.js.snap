// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Block block edit matches snapshot 1`] = `
<Fragment>
  <BlockControls>
    <AlignmentToolbar
      onChange={[Function]}
      value="center"
    />
  </BlockControls>
  <InspectorControls>
    <PanelBody>
      <RangeControl
        label="Height"
        max="10"
        min="1"
        onChange={[Function]}
        value={1}
      />
      <RangeControl
        label="Width"
        max="100"
        min="10"
        onChange={[Function]}
        step="0.1"
        value={50}
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Divider Color",
            "onChange": [Function],
            "value": "#dddddd",
          },
        ]
      }
      title="Color Settings"
    />
  </InspectorControls>
  <div
    className="ugb-divider"
    style={
      Object {
        "paddingBottom": 8,
        "paddingTop": 8,
      }
    }
  >
    <hr
      align="center"
      style={
        Object {
          "backgroundColor": "#dddddd",
          "height": 1,
          "marginBottom": 0,
          "marginTop": 0,
          "width": "50%",
        }
      }
    />
  </div>
</Fragment>
`;

exports[`Divider Block block save matches snapshot 1`] = `
<div
  className="ugb-divider"
>
  <hr
    align="center"
    style={
      Object {
        "backgroundColor": "#dddddd",
        "height": 1,
        "width": "50%",
      }
    }
  />
</div>
`;
